src/app/(app)/(library)/library/[productId]/page.tsx

```
@@ -28,7 +29,9 @@ const page = async ({ params }: Props) => {
   );
   return;
   <HydrationBoundary state={dehydrate(queryClient)}>
-    <ProductView productId={productId} />
+    <Suspense fallback={<p>Loading..</p>}>
+      <ProductView productId={productId} />
+    </Suspense>
   </HydrationBoundary>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)